corehq/apps/reports/scheduled.py

```diff
@@ -1,11 +1,15 @@
 from calendar import monthrange
 from datetime import datetime
 from corehq.apps.reports.models import ReportNotification
+from corehq.util.soft_assert import soft_assert
+
+
+_soft_assert = soft_assert(exponential_backoff=True)
 
 
 def get_scheduled_reports(period, as_of=None):
     as_of = as_of or datetime.utcnow()
-    assert period in ('daily', 'weekly', 'monthly')
+    assert period in ('daily', 'weekly', 'monthly'), period
```

Comment:
@millerdev pr followup 2 years after the fact :) https://github.com/dimagi/commcare-hq/pull/4285/files#r17305879